src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixinclude

```diff
@@ -31,5 +31,8 @@ NuGet.Versioning.dll
 
 # 3rd party dlls to keep
 Lucene.Net.dll
+Microsoft.Extensions.FileProviders.Abstractions.dll
```

Comment:
I have my reservations about this. 
Do we really need these?
No one else in the product brings them?

Reply:
Also you need to make sure these are signed if we are going to pack them in the VSIX.

Reply:
@nkolev92 I just tested this. The three of them are used when adding contentFiles during restore. None of them are brought in by anything else, therefore if we want to use them we need to add them to the VSIX ourselves.

Reply:
Ok, so we know what we'd need to do there. 

How about the dotnet.exe? 
Are they in the SDK already? Those packages would need to be source-build-able.

Reply:
being part of asp.net I've no reason to believe that these packages aren't source-buildable.

Reply:
They are not in the SDK alrady and they are also not source-build-able... There's a plan to add the in .NET Core 3.0